src/main/java/com/fnz/TimeTracking/model/Pointage.java

```diff
@@ -0,0 +1,24 @@
+package com.fnz.TimeTracking.model;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+import java.time.LocalDateTime;
+
+
+@Getter
+@Setter
+@Entity
+public class Pointage {
+    @Id
+    @GeneratedValue
+    private long pointage_id;
+    private String type;
+    private String workType;
+    private LocalDateTime clockIn;
+    private LocalDateTime clockOut;
+
+
```

Comment:
In the provided class diagram, this class should have 3 direct relations:

Pointage > Utilisateur: ManyToOne
Pointage > Depart: OneToOne
Pointage > Arrivée: OneToOne